py/makeversionhdr.py

```diff
@@ -57,7 +57,7 @@ def get_version_info_from_git():
     return git_tag, git_hash, ver
 
 def get_version_info_from_docs_conf():
-    with open("%s/docs/conf.py" % sys.argv[0].rsplit("/", 2)[0]) as f:
+    with open(os.path.join(sys.argv[0].rsplit(os.sep, 2)[0], "docs", "conf.py")) as f:
```

Comment:
This will still fail if a path with forward slashes is passed to the script. As @pfalcon mentioned earlier, the correct way is using os.path.dirname()